docs/htdocs/info/docs/tools/vep/vep_formats.html

```diff
@@ -303,7 +303,16 @@ <h2 id="id"> Variant identifiers</h2>
     the Ensembl Variation database. See <a
     href="/info/genome/variation/species/sources_documentation.html">here</a> for a list of
     identifier sources in Ensembl.</p>
-    
+   
+    <div>
+      <div class="info" style="float:left">
+        <h3>Note</h3>
+        <div class="message-pad">
+          <p>VEP is optimised for the analysis of variants in sorted VCF files or other position based formats, so input in these formats should be in preference to variant names where possible. Using variant names can take 2-5 times longer.</p>
```

Comment:
Slight change:

so input in these formats should be in preference to variant names where possible
->
so input in these formats is recommended to variant names where possible

Reply:
Updated!